foundry/applications/formApplications/sceneConfig.d.ts

```diff
@@ -0,0 +1,30 @@
+/**
```

Comment:
Your version of SceneConfig is missing some methods and in general seems less complete. Unless you specifically need it for your other updates, it is probably best to just remove it or only create it as a stub. Oh and it is in the wrong location as well. It should be in `foundry/applications/formApplications/baseEntitySheets/sceneConfig.d.ts`